src/pages/sweepstake/mydonations/index.jsx

```diff
@@ -0,0 +1,357 @@
+                if(sweepstakeResult[9] == false && sweepstakeResult[10] == false){     
+                    if(sweepstakesArray.includes(sweepstakeResult[3])) continue
+                    sweepstakesArray.push(sweepstakeResult[3])
+                } 
+            }
+    
+            // Get balanceOf() of user in all tokens sweepstakes
+            let balanceOfTokens = []
+            for(const sweepstakeToken of sweepstakesArray){                
+                const balance = await contractSweepstake.methods.balanceOf(address, sweepstakeToken).call();
+                if(ethers.utils.formatEther(balance) > 0){
+                    balanceOfTokens.push([sweepstakeToken, balance])
+                }
+            }
+            setTokensToWithDraw(balanceOfTokens)
+          })
+        }catch(e){
+            console.log(e)
+        }          
+    }
+
+    function renderCampaings(campaing){
+        if(campaing == 'active'){
+            setShowActiveCampaings(true)
+            setShowCompletedCampaings(false)            
+            setShowCanceledCampaings(false)
+            setShowWinsCampaings(false)
+        }
+        if(campaing == 'completed'){
+            setShowCompletedCampaings(true)            
+            setShowActiveCampaings(false)
+            setShowCanceledCampaings(false)
+            setShowWinsCampaings(false)
+        }
+        if(campaing == 'wins'){
+            setShowWinsCampaings(true)
+            setShowActiveCampaings(false)
+            setShowCompletedCampaings(false)            
+            setShowCanceledCampaings(false)
+        }
+        if(campaing == 'canceled'){
+            setShowCanceledCampaings(true)
+            setShowActiveCampaings(false)
+            setShowCompletedCampaings(false)            
+            setShowWinsCampaings(false)
+        }        
+    }
+
+    function card(
+        nftID,
+        nftTokenURI,
+        nftOwner,
+        nftErc20token,
+        nftDuration,
+        nftTotalStaked,
+        nftWinner,
+        nftDrawTimestamp,
+        nftActive,
+        nftDestroyed
+    ){
+        return (
+            <div className="card my-3">
+                <div className="card-body">
+                    <h2 className="card-title">NFT ID: {nftID}</h2>
+                    <div>
+                        <span className="font-bold mr-3">Owner:</span> {nftOwner}</div>
+                    <div>
+                        <span className="font-bold mr-3">Token:</span>
+                        {
+                        nftErc20token == process.env.NEXT_PUBLIC_CBRL ? `${nftErc20token} (pBRL)` :
+                        nftErc20token == process.env.NEXT_PUBLIC_CUSD ? `${nftErc20token} (pUSD)` :
+                        nftErc20token == process.env.NEXT_PUBLIC_CEUR ? `${nftErc20token} (pEUR)` :
+                        null
+                        }
+                    </div>
+                    { false == true ?
```

Comment:
Codacy has a fix for the issue: [Expected '===' and instead saw '=='.](https://app.codacy.com/gh/esolidar/web3/pullRequest?prid=9542591)
```suggestion
                    { false === true ?
```